src/sentry/grouping/api.py

```diff
@@ -141,6 +142,8 @@ def _get_calculated_grouping_variants_for_event(event, config):
 
     for strategy in config.iter_strategies():
         rv = strategy.get_grouping_component_variants(event, config=config)
+        rv = collections.OrderedDict(sorted(rv.items()))
```

Comment:
We could potentially move the `OrderedDict` into the `get_grouping_component_variants` to avoid the `sorted`